scripts/e2e/delete_fixtures

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash -eu
+
+# Does the inverse of create_fixtures
+echo && echo "[$0] deleting fixtures for api e2e tests..."
+
+delete_doc() {
+	local docUrl="$COUCH_URL/$1"
+	local rev="$(curl $docUrl | jq --raw-output ._rev)"
+	if [[ "$rev" != "null" ]]; then
+		curl -X DELETE "$docUrl?rev=$rev"
+	fi
+}
+
+delete_user() {
+	local username="$1"
+	echo "$API_URL/api/v1/users/$1"
+	curl -X DELETE "$API_URL/api/v1/users/$1"
+}
+
+echo "[$0] deleting users created by test fixtures..."
```

Comment:
Why only 1 `echo` here?  (Not that I can remember why there were 2 elsewhere...)